tests/test_router_bgp.rb

```diff
@@ -580,9 +580,99 @@ def test_default_enforce_first_as
     bgp.destroy
   end
 
-  def test_event_history
+  def test_event_history_cli_evergreen
```

Comment:
I would get ride of the `_evergreen` bit and just make sure this test tests for event_history_cli functionality regardless of version.  This of course means you do version checks in the test case.